lib/rules/no-aggregating-enable.js

```diff
@@ -26,24 +26,20 @@ module.exports = {
     },
 
     create(context) {
-        const sourceCode = context.getSourceCode()
-        const disabledArea = DisabledArea.get(sourceCode)
+        const disabledArea = getDisabledArea(context)
 
-        return {
-            Program() {
-                for (const entry of disabledArea.numberOfRelatedDisableDirectives) {
-                    const comment = entry[0]
-                    const count = entry[1]
+        for (const entry of disabledArea.numberOfRelatedDisableDirectives) {
+            const comment = entry[0]
+            const count = entry[1]
 
-                    if (count >= 2) {
-                        context.report({
-                            loc: utils.toForceLocation(comment.loc),
-                            messageId: "aggregatingEnable",
-                            data: { count },
-                        })
-                    }
-                }
-            },
+            if (count >= 2) {
+                context.report({
+                    loc: utils.toForceLocation(utils.getLoc(context, comment)),
+                    messageId: "aggregatingEnable",
+                    data: { count },
+                })
+            }
         }
```

Comment:
```suggestion
        disabledArea.numberOfRelatedDisableDirectives
            .filter(([, count]) => count >= 2)
            .forEach(([comment, count]) => context.report({
                loc: utils.toForceLocation(utils.getLoc(context, comment)),
                messageId: "aggregatingEnable",
                data: { count },
            }))
```

Reply:
This part respects the original implementation, and I think it's better to make a separate PR if we want to change it.